internal/pkg/service/buffer/storage/local/writer/disksync/disksync.go

```diff
@@ -279,11 +280,13 @@ func (s *Syncer) syncLoop() {
 				// The Close method has been called
 				return
 			case <-ticker.C:
-				countTrigger := s.writeOpsCount.Load() >= uint64(s.config.CountTrigger)
-				bytesTrigger := datasize.ByteSize(s.bytesToSync.Load()) >= s.config.BytesTrigger
-				intervalTrigger := s.clock.Now().Sub(s.lastSyncAt.Load()) >= s.config.IntervalTrigger
-				if countTrigger || bytesTrigger || intervalTrigger {
-					s.TriggerSync(false)
+				if count := s.writeOpsCount.Load(); count > 0 {
+					countTrigger := count >= uint64(s.config.CountTrigger)
+					bytesTrigger := datasize.ByteSize(s.bytesToSync.Load()) >= s.config.BytesTrigger
+					intervalTrigger := s.clock.Now().Sub(s.lastSyncAt.Load()) >= s.config.IntervalTrigger
+					if countTrigger || bytesTrigger || intervalTrigger {
+						s.TriggerSync(false)
+					}
```

Comment:
Na zaver som pridal spat este podmienku, ktora bola predtym `bytes > 0`, ale bolo to nepresne, pretoze CSV writer ma este interny buffer, ... nahradene za `count > 0`, ktory sa reportuje presne.

Reply:
Pomoze nam to na multitenant stacks, `receivers`, ktore su malo vytazene nebudu zbytocne spustat `sync` kazdych `50ms`, ak nedostali ziadne nove data.